mozdownload/scraper.py

```diff
@@ -51,6 +51,9 @@
                            'win32': 'exe',
                            'win64': 'exe'}
 
+DEFAULT_BRANCHES = {'thunderbird': 'comm-central'
```

Comment:
Both have `mozilla-central`, as you hard-coded below. So just move it up here and do not fallback to a default for an unknown version.